test/common/db_sandbox.js

```diff
@@ -49,6 +49,7 @@ function DBSandbox(dbConfig, testDatabaseName) {
 	this.originalDatabaseName = dbConfig.database;
 	this.testDatabaseName = testDatabaseName || this.originalDatabaseName;
 	this.dbConfig.database = this.testDatabaseName;
+	this.dbConfig.noLocking = true;
```

Comment:
There is already `initOptions.noLocking = true;` in the same file. (https://github.com/LiskHQ/lisk/pull/1739/files#diff-adf057742c94318bbf06eb7ee29a72baR25). One of the `noLocking = true` assignment is definitely redundant. Please remove the global one if possible, or do not introduce the new assignment otherwise.